src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingUtils.java

```diff
@@ -237,18 +240,22 @@ public static ImmutableMap<String, String> validateStreamingSnowpipeConfig(
             BOOLEAN_VALIDATOR.ensureValid(
                 ERRORS_LOG_ENABLE_CONFIG, inputConfig.get(ERRORS_LOG_ENABLE_CONFIG));
           }
+          if (inputConfig.containsKey(ENABLE_CHANNEL_OFFSET_TOKEN_MIGRATION_CONFIG)) {
```

Comment:
see comment in https://github.com/snowflakedb/snowflake-kafka-connector/pull/822/files#r1568735719